compiler/noirc_evaluator/src/ssa/plonky2_gen/intrinsics/sha256.rs

```diff
@@ -48,8 +47,8 @@ fn array_to_bits(bytes: &[u8]) -> Vec<bool> {
     ret
 }
 
-fn u32_to_bits_target<F: RichField + Extendable<D>, const D: usize, const B: usize>(
-    builder: &mut CircuitBuilder<F, D>,
+fn u32_to_bits_target<const B: usize>(
+    builder: &mut impl AsmWriter,
```

Comment:
nit: rename `builder` to `asm_writer`

Reply:
Done: https://github.com/blocksense-network/noir/pull/25/commits/7e9ec7c0194eb3dd549d940f33c6f1ba16364754